src/corehost/cli/fx_reference.cpp

```diff
@@ -9,8 +9,8 @@
 
 bool fx_reference_t::is_roll_forward_compatible(const fx_ver_t& other) const
 {
-    // We expect the version to be <
-    assert(get_fx_version_number() < other);
+    // We expect the version to be <=
+    assert(get_fx_version_number() <= other);
```

Comment:
If the function was renamed `is_compatible` we would remove these asserts and add 

```C#
if (get_fx_version_number() > other) other.is_compatible(this);
```

Reply:
It's just never used like that (the caller needs to make the distinction between the two cases for other reasons anyway).

I renamed the method to `is_compatible_with_higher_version` to make it explicit.